core/src/test/scala/org/apache/spark/scheduler/MapStatusSuite.scala

```diff
@@ -188,4 +188,37 @@ class MapStatusSuite extends SparkFunSuite {
       assert(count === 3000)
     }
   }
+
+  test("YSPARK-500 MapStatus has 2000 hardcoded") {
```

Comment:
rename to "HighlyCompressedMapStatus has configurable threshold"

Reply:
there's also a typo in the bug number.